src/constants/constants.js

```diff
@@ -24,4 +24,5 @@ export const JSFEAT_CONSTANTS = {
     // svd options
     SVD_U_T: 0x01,
     SVD_V_T: 0x02,
+    U8C1_t: 0x0100 | 0x01,
```

Comment:
because in te ransac method from motion_estimator U8C1_t is instantiated fro JSFEAT_CONSTANTS:
https://github.com/kalwalt/jsfeatNext/blob/ee19e33065c7adfbeba7d13092cce552ae83ce78/src/motion_estimator/motion_estimator.js#L76
and that doesn't exist this was causing the issue in the sample_orb example. Declaring in the constants solved that.

Reply:
I will add also the other missed.